Source/Initialization/WarpXInitData.cpp

```diff
@@ -420,10 +441,111 @@ WarpX::InitLevelData (int lev, Real /*time*/)
     }
 
 #ifdef WARPX_MAG_LLG
+    // if the input string for the Hbias-field is "parse_h_bias_ext_grid_function",
+    // then the analytical expression or function must be
+    // provided in the input file.
+    if (H_bias_ext_grid_s == "parse_h_bias_ext_grid_function") {
```

Comment:
add this input option in the docs as well. WarpX/docs/running_cpp/parameters.rst